influunt-api/conf/evolutions/default/2.sql

```diff
@@ -28,3 +31,8 @@ ALTER TABLE `controladores_fisicos` DROP COLUMN `area_id`;
 
 ALTER TABLE sessoes DROP COLUMN ip;
 ALTER TABLE sessoes DROP COLUMN data_saida;
+ALTER TABLE sessoes DROP COLUMN ip;
+
```

Comment:
O 2.sql não é para ser alterado, deve ser criado o 3.sql. Se for feito assim, o play vai primeiro rodar o downs do 2.sql (apagando dados no processo), e depois rodar o 2.sql de novo. Se fizer o 3.sql ele não vai desfazer o 2.sql.